.github/workflows/build.yml

```
@@ -199,6 +199,46 @@ jobs:
           name: coverage
           path: ./coverage
 
+  test-freebsd-amd64:
+    runs-on: macos-10.15
```

```suggestion
    runs-on: macos-12
```

macos-10.15 instances will be shut down soon
- https://github.com/actions/virtual-environments/issues/5583

---

Thanks!